src/axom/primal/operators/detail/intersect_impl.hpp

```diff
@@ -381,36 +386,20 @@ inline bool intersectTwoPermutedTriangles(const Point3& p1,
                                           const Point3& p2,
                                           const Point3& q2,
                                           const Point3& r2,
-                                          bool includeBoundary)
+                                          bool includeBoundary,
+                                          double EPS)
 {
   /* Step 5: From step's 1 through 4, we now have two triangles that,
      if intersecting, have a line that intersects segments p1r1, p1q1,
      p2q2, and p2r2.  We check if these two intervals overlap:
    */
-
-  if (!isLpeq(Vector3(q1, q2).dot(Triangle3(q1, p2, p1).normal()), 0.0,
-              includeBoundary) ||
-      !isLpeq(Vector3(p1, r2).dot(Triangle3(p1, p2, r1).normal()), 0.0,
-              includeBoundary))
-  {
-    return false;
-  }
-
-  return true;
+  const bool bdr = includeBoundary;
+  /* *INDENT-OFF* */
+  return isLpeq(Vector3(q1, q2).dot(Triangle3(q1, p2, p1).normal()), 0.0, bdr, EPS)
+      && isLpeq(Vector3(p1, r2).dot(Triangle3(p1, p2, r1).normal()), 0.0, bdr, EPS);
+  /* *INDENT-ON* */
```

Comment:
Here is the key change -- we need to get the user's `EPS` value down to this function.